browser/metrics/metrics_reporting_util.h

```diff
@@ -0,0 +1,11 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_METRICS_METRICS_UTIL_H_
```

Comment:
nit: header guard doesn’t match file path

Reply:
Oops! I forgot to run lint again after renaming.
Done.